course_content/notebooks/matplotlib_intro.ipynb

```diff
@@ -81,25 +68,15 @@
-       "text": [
-        "<matplotlib.figure.Figure at 0x43c21d0>"
-       ]
-      }
-     ],
-     "prompt_number": 3
+     "outputs": []
```

Comment:
I think you may be right as far as the static notebooks are concerned.  I generally work to the principle that when writing notebooks and committing to source control, it is always best to clear all output for size and simplicity purposes, but maybe that's not the case here.

Would it be better to create the static notebooks for the Readme with cell outputs, but have clear outputs for the live notebooks.  After all, in the course itself, all the notebooks' cells will be executed in order.